posthog/models/property_definition.py

```diff
@@ -5,6 +5,16 @@
 from posthog.models.utils import UUIDModel
 
 
+class PropertyType(models.TextChoices):
+    Datetime = "DateTime", "DateTime"
```

Comment:
Should we already add more values here? Adding enums later in django requires another db migration to be made and run.

Reply:
I was wondering about this... I was anticipating the migration would be cheap to run (drop and add a constraint) Am expecting that I'll add more before proper release. Wanted to get DateTime in so I can prove the shape without thinking about too much at once

Reply:
I guess it'll be fine. The table we're operating on won't have *that* many rows in the end (few million max?), so even if it's a more expensive index query (doubtful in the case of a constraint?), I think we're fine. Though I've been burned by postgres migrations more times than I can count, so take from this what you want :D.

Reply:
Yeah, no harm in adding string, number, bigint, and boolean to match https://github.com/PostHog/posthog/blob/master/frontend/src/lib/components/PropertiesTable.tsx#L12

As things stand we'd need a migration each time we add a format to update the check constraint

Reply:
@mariusandra added the types to bring the format in-line with the types that the `PropertiesTable` knows how to display